res/skins/QMLDemo/WaveformShader.qml

```diff
@@ -0,0 +1,222 @@
+import Mixxx 0.1 as Mixxx
+import QtQuick 2.12
+
+ShaderEffect {
+    id: root
+
+    property string group // required
+    property var deckPlayer: Mixxx.PlayerManager.getPlayer(group)
+    property variant framebufferSize: Qt.size(width, height)
```

Comment:
`variant` in considered obsolete:
https://doc.qt.io/qt-5/qml-variant.html (first sentence)